internal/env/scm.go

```diff
@@ -104,3 +104,11 @@ func HasAppID() bool {
 	_, err := AppID()
 	return err == nil
 }
+
+func GetAppURL() string {
+	return os.Getenv("QUICKFEED_APP_URL")
```

Comment:
We currently only load the root `.env` file:

```go
// Load environment variables from $QUICKFEED/.env.
// Will not override variables already defined in the environment.
const envFile = ".env"
if err := env.Load(env.RootEnv(envFile)); err != nil {
	log.Fatal(err)
}
```

The `QUICKFEED_APP_URL` is written to `/public/.env`, and is not loaded into the environment at any point.

```go
publicEnvFile := env.PublicEnv(m.envFile)
if err := env.Save(publicEnvFile, map[string]string{
	"QUICKFEED_APP_URL": config.GetHTMLURL(),
}); err != nil {
	return err
}
```

I think this will return an empty string.

Reply:
We likely don't need the `/public/.env` file anymore as we no longer use webpack.
The app URL can be saved in the root `.env` file.